docs/examples/models/isotropic_thin_3d.py

```diff
@@ -16,19 +17,28 @@
     "wavelength_illumination": 0.532,
     "index_of_refraction_media": 1.3,
 }
-phantom_arguments = {"index_of_refraction_sample": 1.33, "sphere_radius": 5}
+phantom_arguments = {"index_of_refraction_sample": 1.50, "sphere_radius": 5}
```

Comment:
Is this still a 'weak object'?

Reply:
fixed so that absorption and phase have the same strength